### PR TITLE
[W5.11][W13-A3]Jarrett Choo

### PR DESCRIPTION
**Pull request by Kyomian**
Added sort command to Address Book.

Updated I/O tests. But runtests/Travis gave me error, even though both actual.txt and expected.txt are the same.

### CODE REVIEW

**Review by bayweiheng — CHANGES_REQUESTED** [review]
Person.java: just return the compareTo result directly, there is no need for the if-elseif-else

File: src/seedu/addressbook/commands/Command.java [hunk 1]
```diff
@@ -36,6 +36,10 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
         return String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, personsDisplayed.size());
     }
 
+    public static String getMessageForPersonSortedShownSummary(List<? extends ReadOnlyPerson> personsDisplayed) {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/commands/Command.java:39]: Do you think that changing the method name to `getSummaryForSortedPersons` is clearer?

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import java.util.List;
+
+/**
+ * Sorts the address book
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/commands/SortCommand.java:7]: The header comment should provide the usage of the class, in this case can we be more detailed and precise with what this class is supposed to handle =)? Is the class supposed to `Sorts persons in the address book, according to their names in lexicographical order`?

File: src/seedu/addressbook/data/person/Person.java [hunk 3]
```diff
@@ -84,4 +84,21 @@ public String toString() {
...
+    /**
+     * Compares Person objects, according to Person names
+     * @param other Other Person object
+     * @return a negative integer, zero, or a positive integer as
+     * this Person is less than, equal to, or greater than the other Person
+     */
+    public int compareTo(Person other) {
+        String thisPersonName = this.getName().toString();
+        String otherPersonName = other.getName().toString();
+        if (thisPersonName.compareTo(otherPersonName) > 0)
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Person.java:96]: [Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. brace style problem.

**Review by brandonyeoxg — COMMENTED** [review]
@Kyomian Some comments added. Please close the PR after reading comments. 

Hey Jarrett I believe that you have to do 2 enhancements for this LO instead of 1...
You can work till this coming tutorial to finish the remaining enhancement =).

File: src/seedu/addressbook/commands/Command.java (hunk 1 above)
Comment (Kyomian) [reply src/seedu/addressbook/commands/Command.java:39]: Yes

File: src/seedu/addressbook/commands/SortCommand.java (hunk 2 above)
Comment (Kyomian) [reply src/seedu/addressbook/commands/SortCommand.java:7]: Yup

**Pull request closed without merging**